src/databricks/labs/ucx/workspace_access/groups.py

```diff
@@ -478,7 +477,7 @@ def _list_account_groups(self, scim_attributes: str) -> list[iam.Group]:
         sorted_groups: list[iam.Group] = sorted(account_groups, key=lambda _: _.display_name)  # type: ignore[arg-type,return-value]
         return sorted_groups
 
-    @retried(on=[DatabricksError])
+    @retried(on=retryable_exceptions)
```

Comment:
here, it'll make sense to retry on InternalError, ResourceConflict, DeadlineExceeded, but not all errors